utils/client_server_name.js

```diff
@@ -61,6 +61,7 @@ const clientToServerNames = {
   JUNE: 'June',
   ENGAGE: 'Engage',
   ITERABLE: 'Iterable',
+  YANDEX_METRICA: 'YandexMetrica',
```

Comment:
Is `YandexMetrica` the display name?

Reply:
No, It's Yandex.Metrica

Reply:
client to server name mapping means config backend name to display name. Here you should put the display name.

Reply:
Okay, Correcting it.

Reply:
Done